child_compassion/wizard/child_description_wizard.py

```diff
@@ -0,0 +1,167 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2014-2015 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: Cyril Sester, Kevin Cristi, David Coninckx
+#
+#    The licence is in the file __openerp__.py
+#
+##############################################################################
+
+from openerp import models, fields, api, exceptions, _
+from child_description_fr import Child_description_fr
+from child_description_de import Child_description_de
+from child_description_en import Child_description_en
+from child_description_it import Child_description_it
+import re
+
+
+class child_description_wizard(models.TransientModel):
+    _name = 'child.description.wizard'
+
+    ##########################################################################
+    #                                 FIELDS                                 #
+    ##########################################################################
+    child_id = fields.Many2one(
+        'compassion.child', 'Child',
+        default=lambda self: self.env.context.get('child_id'))
+    keep_desc_fr = fields.Boolean('Update french description')
+    desc_fr = fields.Html(
+        'French description', default=lambda self: self._get_desc('fr'))
+    keep_desc_de = fields.Boolean('Update german description')
+    desc_de = fields.Html(
+        'German description', default=lambda self: self._get_desc('de'))
+    keep_desc_it = fields.Boolean('Update italian description')
+    desc_it = fields.Html(
+        'Italian description', default=lambda self: self._get_desc('it'))
+    keep_desc_en = fields.Boolean('Update english description')
+    desc_en = fields.Html(
+        'English description', default=lambda self: self._get_desc('en'))
+    child_property_value_ids = fields.Many2many(
+        'compassion.translated.value', compute='_get_value_ids',
+        inverse='_write_values', default=lambda self: self._get_value_ids())
+    comments = fields.Text(
+        'Comments', compute='_get_comments',
+        default=lambda self: self._get_comments(), readonly=True)
```

Comment:
Is it necessary to use a `lambda` function? Call `_get_comments()` isn't adequate?